cmapi/mcs_cluster_tool/cluster_app.py

```diff
@@ -32,9 +44,119 @@ def status():
 
 @app.command()
 @handle_output
-def stop():
+@TransactionManager(
+    timeout=timedelta(days=1).total_seconds(), handle_signals=True
+)
+def stop(
+    interactive: Annotated[
+        bool,
+        typer.Option(
+            '--interactive/--no-interactive', '-i/-no-i',
+            help=(
+                'Use this option on a highly loaded cluster. '
```

Comment:
How about this? if accurate
**Use this option on a active cluster as interactive stop waits for current writes to complete in DMLProc before shutting down. Ensuring consistency, preventing data loss of active writes.**
or shorter
Ensures consistency and prevents data loss by waiting for active writes.

Reply:
First one is marvelous!

@mariadb-AllenHerrera but I'm confused about `a active cluster` not `an active cluster`. Why?

Reply:
Just remove `a` adjective allows one to drop article.

Reply:
Done.